annotation-processor/src/main/kotlin/pl/touk/krush/source/MappingsGenerator.kt

```diff
@@ -84,6 +95,14 @@ class MappingsGenerator : SourceGenerator {
     }
 
     private fun buildToEntityMapFunc(entityType: TypeElement, entity: EntityDefinition, graphs: EntityGraphs): FunSpec {
+        return if(generateRealReferences){
```

Comment:
maybe it would be better to do it by inheritance? `buildToEntityMapFunc` would be abstract and we would have 2 implementations of `MappingsGenerator` - `CopiedReferencesMappingsGenerator` and `RealReferencesMappingsGenerator`

Reply:
Hm, I feel like creating two whole new classes for one changed method is a bit much. Well, it's your call if you prefer it this way ^^